pkg/machine/qemu/machine.go

```diff
@@ -407,7 +411,35 @@ func (v *MachineVM) Init(opts machine.InitOptions) (bool, error) {
 		WritePath: v.getIgnitionFile(),
 		UID:       v.UID,
 	}
+
 	err = machine.NewIgnitionFile(ign)
+	if err == nil {
+		// write to known_hosts
+		hd := homedir.Get()
+		known := filepath.Join(hd, ".ssh", "known_hosts")
+		f, err := os.OpenFile(known, os.O_APPEND|os.O_WRONLY, 0600)
+		if err != nil {
+			if os.IsNotExist(err) {
+				logrus.Warn("podman machine will soon require a known_hosts file to function properly. Podman will create and mamage keys within ~/.ssh/known_hosts")
```

Comment:
I don't like the "will soon" part of this message,  Either it requires it or it does not.